gcp/table_gcp_compute_tpu.go

```diff
@@ -0,0 +1,252 @@
+func tableGcpComputeTpu(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "gcp_compute_tpu",
+		Description: "GCP Compute TPUs are specialized hardware accelerators designed to speed up specific machine learning workloads.",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			Hydrate:    getComputeTpu,
+			Tags:       map[string]string{"service": "tpu", "action": "GetNode"},
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listComputeTpus,
+			Tags:    map[string]string{"service": "tpu", "action": "ListNodes"},
+		},
+		GetMatrixItemFunc: BuildTpuZoneList,
+		Columns: []*plugin.Column{
+			// Key columns
+			{
+				Name:        "name",
+				Description: "The name of the TPU node.",
+				Type:        proto.ColumnType_STRING,
+			},
+
+			// Other columns
+			{
+				Name:        "id",
+				Description: "The unique identifier for the TPU node.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "description",
+				Description: "The user-supplied description of the TPU node.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "accelerator_type",
+				Description: "The type of TPU accelerator.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "state",
+				Description: "The current state of the TPU node.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "health_description",
+				Description: "If the TPU node is unhealthy, this contains more detailed information about why.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "network",
+				Description: "The name of the network that the TPU node is connected to.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "cidr_block",
+				Description: "The CIDR block that the TPU node will use when selecting an IP address.",
+				Type:        proto.ColumnType_CIDR,
+			},
+			{
+				Name:        "service_account",
+				Description: "The service account used to run the TPU node.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "create_time",
+				Description: "The time when the TPU node was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "scheduling_config",
+				Description: "Sets the scheduling options for the TPU instance.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "network_endpoints",
+				Description: "The network endpoints where the TPU node can be accessed.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "labels",
+				Description: "Resource labels to represent user provided metadata.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "zone",
+				Description: "The GCP zone where the TPU node is located.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(gcpTpuTurbotData, "Zone"),
+			},
+
+			// Tags
+			{
+				Name:        "tags_src",
```

Comment:
We don't include the `tags_src` column in any of the other tables. Could you please remove it from the table schema?